### PR TITLE
added periodic method to indexer

### PR DESCRIPTION
**Pull request by MarcusPickett**
added periodic method to indexer

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Indexer.java [hunk 1]
```diff
@@ -13,26 +13,31 @@
...
-        conveyor.set(ControlMode.PercentOutput,-speed);
+
+    public void reverseFeedShooter() {
+        conveyor.set(ControlMode.PercentOutput, -speed);
     }
     // System.out.print("we need a sensor here");
+    @Override
+  public void periodic() {
+    // This method will be called once per scheduler run
+  }
```
Comment (mott-lab) [inline src/main/java/frc/robot/subsystems/Indexer.java:47]: These lines should be indented.

File: src/main/java/frc/robot/subsystems/Indexer.java [hunk 2]
```diff
@@ -13,26 +13,31 @@
...
-    final double speed =  1;
-    VictorSPX cellHolder1=new VictorSPX(1);
+public class Indexer extends SubsystemBase {
+    final double speed = 1;
+    VictorSPX cellHolder1 = new VictorSPX(1);
     VictorSPX cellHodler2 = new VictorSPX(2);
-    VictorSPX conveyor= new VictorSPX(3);
-    Solenoid cellGate= new Solenoid(4);
+    VictorSPX conveyor = new VictorSPX(3);
+    Solenoid cellGate = new Solenoid(4);
```
Comment (duffym22) [inline src/main/java/frc/robot/subsystems/Indexer.java]: Replace magic numbers and reference existing CAN ID constants in Constants.java

**Review by duffym22 — CHANGES_REQUESTED** [review]
See comments

File: src/main/java/frc/robot/subsystems/Indexer.java [hunk 3]
```diff
@@ -12,13 +12,14 @@
...
 public class Indexer extends SubsystemBase {
     final double speed = 1;
-    VictorSPX cellHolder1 = new VictorSPX(1);
-    VictorSPX cellHodler2 = new VictorSPX(2);
-    VictorSPX conveyor = new VictorSPX(3);
-    Solenoid cellGate = new Solenoid(4);
+    VictorSPX cellHolder1 = new VictorSPX(Constants.INDEXER_CELLHOLDER1_ID);
+    VictorSPX cellHodler2 = new VictorSPX(Constants.INDEXER_CELLHOLDER2_ID);
+    VictorSPX conveyor = new VictorSPX(Constants.INDEXER_CONVEYER_ID);
+    Solenoid cellGate = new Solenoid(Constants.INDEXER_CELLGATE_ID);
```
Comment (borkae) [inline src/main/java/frc/robot/subsystems/Indexer.java]: Add private to the class variables and move "new" operations into constructor.

**Review by borkae — APPROVED** [review]
Recommended changes complete.

**Review by duffym22 — APPROVED** [review]
Approved

**Pull request merged**